src/filters/trace.rs

```diff
@@ -46,12 +46,17 @@ pub fn request() -> Trace<impl Fn(Info) -> Span + Clone> {
             "request",
             remote.addr = Empty,
             method = %info.method(),
+            host = Empty,
```

Comment:
Is it generally useful to always include the host? I'd imagine for most servers, the actual host string isn't interesting, and someone would only want to see it if they served multiple hosts on the same server.

Reply:
That's true, on the other hand for me host is, but I really don't care about version and proto, for example.

I would include it since in multi-host setups path is not interpretable without it, but if we don't want to, not a big deal either, as the `Fn(Info) -> Span` function can be defined in own code. Let me know.